doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -133,17 +135,26 @@ size isn't smaller than 4% of ``block``, because RGW heavily uses it to store it
 metadata. For example, if the ``block`` size is 1TB, then ``block.db`` shouldn't
 be less than 40GB. For RBD workloads, 1% to 2% of ``block`` size is usually enough.
 
-If *not* using a mix of fast and slow devices, it isn't required to create
-separate logical volumes for ``block.db`` (or ``block.wal``). Bluestore will
-automatically manage these within the space of ``block``.
+In releases prior to Pacific, DB internal level sizes meant that the DB could
+only take advantage of specific partition / LV sizes that correspond to sums of
+L0, L0+L1, L1+L2, etc., which with default settings means roughly 3 GB, 30 GB,
+30 0GB, and so forth.  Most deployments will not substantially benefit from
+sizing to accomodate L3 and higher, though DB compaction can be facilitated by
+doubling these figures to 6GB, 60GB, and 600GB.  Improvements in Pacific and
```

Comment:
The latest Octopus and Nautilus benefit from better utilization as well..

Reply:
Pacific likely to get another one improvement (which is using dynamic levels in RocksDB) in this area, see https://github.com/ceph/ceph/pull/37156
But this is going to be experimental for some time..

Reply:
Thanks, Igor. Exactly the input I need here.  Clarity and accuracy are sorely needed here, and I don't want to add in more text that is inaccurate.  I do remember https://github.com/ceph/ceph/pull/29687 but wasn't sure that it actually enabled better utilization vs. providing something upon which to build in the future.

I've reworked this, PTAL.